src/BlockTypes/Cart.php

```diff
@@ -222,9 +222,23 @@ protected function deep_sort_with_accents( $array ) {
 			return $array;
 		}
 
-		if ( is_array( reset( $array ) ) ) {
-			return array_map( [ $this, 'deep_sort_with_accents' ], $array );
-		}
+		$array_without_accents = array_map(
+			'remove_accents',
+			array_map(
+				'wc_strtolower',
+				array_map(
+					'html_entity_decode',
+					array_map(
+						function ( $element ) {
+							if ( is_array( $element ) ) {
+								return $this->deep_sort_with_accents( $element );
+							}
+						},
+						$array
+					)
+				)
+			)
+		);
```

Comment:
I'm not sure I understand why we need this code 🤔 . In line 243 below, we reassign a new value for `$array_without_accents` without using its value here.

Reply:
Just confirmed that, L225 - L241 is dead code right now.

<img width="791" alt="image" src="https://user-images.githubusercontent.com/17236129/195816406-813c8ecb-290d-4d86-8aeb-0932c4ced684.png">